core/src/i18n/en.yaml

```diff
@@ -1720,6 +1721,10 @@ replica-sets:
   tooltips:
     error: At least one of the Pods has failed.
     running: All the Pods are running correctly.
+resource-graph:
+  error: Something went wrong during rendering the graph.
```

Comment:
Can we provide more info to the user what can be done to fix it?

Reply:
I don't think so, it would be more like technical info (call stack, error message) etc.